archeana.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "Ana Jesus",
+  "bio": "Web Developer & Archaeo Enthusiast."
```

Please add the missing comma here! 

```suggestion
  "bio": "Web Developer & Archaeo Enthusiast.",
```